container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend_enclave_functions.go

```diff
@@ -24,6 +25,10 @@ const (
 
 	defaultHttpLogsCollectorPortNum = uint16(9712)
 	defaultTcpLogsCollectorPortNum  = uint16(9713)
+
+	labelTypeName       = "com.kurtosistech.container-type"
+	filterContainerType = "api-container"
```

Comment:
same comment here, you can replace it with this constant: `label_value_consts.apiContainerContainerTypeLabelValueStr`